internal/action/main_test.go

```diff
@@ -66,14 +64,11 @@ func removeTestWorkspaces(t *testing.T, ctx context.Context, client *tfe.Client,
 			PageSize: maxPageSize,
 		},
 	})
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
 
 	for _, ws := range workspaces.Items {
-		if err := client.Workspaces.DeleteByID(ctx, ws.ID); err != nil {
-			t.Fatal(err)
-		}
+		err := client.Workspaces.DeleteByID(ctx, ws.ID)
+		require.NoError(t, err)
```

Comment:
```suggestion
		assert.NoError(t, err)
```

Loop should proceed even if one error occurs